toolchain/driver/clang_runner.cpp

```diff
@@ -484,33 +629,40 @@ auto ClangRunner::BuildBuiltinsFile(llvm::StringRef target,
 auto ClangRunner::BuildBuiltinsLib(llvm::StringRef target,
                                    const llvm::Triple& target_triple,
                                    const std::filesystem::path& tmp_path,
-                                   Filesystem::DirRef lib_dir)
+                                   Filesystem::DirRef lib_dir,
+                                   llvm::ThreadPoolInterface& threads)
     -> ErrorOr<Success> {
   llvm::SmallVector<llvm::StringRef> src_files =
       CollectBuiltinsSrcFiles(target_triple);
 
   CARBON_ASSIGN_OR_RETURN(Filesystem::Dir tmp_dir,
                           Filesystem::Cwd().OpenDir(tmp_path));
 
-  llvm::SmallVector<llvm::NewArchiveMember> objs;
+  std::mutex mu;
```

Comment:
Comment that this guards access of objs? Or maybe put `GUARDED_BY(mu)` on objs?

Do we compile with `-Wthread-safety`?

Reply:
Thanks that looks nicer